src/modules/core/components/ActionsList/ActionsListItem.tsx

```diff
@@ -196,6 +205,7 @@ const ActionsListItem = ({
                 decimals: getTokenDecimalsWithFallback(decimals),
                 fromDomain: domainName || fromDomain?.name || '',
                 toDomain: toDomain?.name || '',
+                roles: role ? getFormattedRole() : '',
               }}
             />
           </div>
```

Comment:
The actions list should filter out the initial permissions, which it still does not:

![Screenshot from 2021-02-10 17-34-02](https://user-images.githubusercontent.com/1193222/107534074-3d3d7080-6bc8-11eb-8b90-78c3b5010d0a.png)